sys/debug/kgmon.py

```diff
@@ -0,0 +1,46 @@
+import gdb
+import sys
+
+
+from .cmd import SimpleCommand
+
+
+class GmonOut():
+
+    def write_to_file(self, path="gmon.out"):
```

Comment:
You could also make this a static method, then you don't have to create an instance to call it, you can just do `GmonOut.write_to_file()`